src/guide/data-methods.md

```diff
@@ -0,0 +1,137 @@
+# Data Properties and Methods
+
+## Data Properties
+
+The `data` function for a component is called as part of creating a new component instance. It should return an object, which Vue wraps in its reactivity system and stores on the instance as `$data`. For convenience, any top-level properties of that object are also exposed directly via the component instance:
```

Comment:
```suggestion
The `data` property for a component is called as part of creating a new component instance. It is a function that should return an object, which Vue will then wrap in its reactivity system and store on the component instance as `$data`. For convenience, any top-level properties of that object are also exposed directly via the component instance:
```

Reply:
Suggested edits are motivated by people not as familiar with JS, so calling it a property would be easier to follow initially, and then defining it technically should allow them that gradual ease into the concepts for data.

Reply:
The later parts are now integrated.

I don't think we can open the paragraph with *'The `data` property'* because that would conflict with the more common meaning of 'data property'.

I've had a go at rewording the start of this paragraph to take things slower.